examples/tools/treemap/readme.md

```diff
@@ -0,0 +1,48 @@
+# Mapping Market Movements with Polygon.io and D3.js Treemap
+
+This repository contains code and resources for visualizing stock market data using D3.js Treemap and Polygon.io's API.
+
```

Comment:
Are you planning on linking the future blog post here? If so, will it be going into more detail about what this treemap represents (ie what does each box represent, what do the colors mean)? If the README explains exactly what we're making it makes reading and understanding the code easier.

I'd also suggest putting the screenshot from the PR in this README to make it more clear what this code does without having to clone and run it.

Reply:
Fixed - added more an intro paragraph, a link to the tutorial, and an image.